src/Components/Books.module.css

```diff
@@ -1,14 +1,11 @@
-.title {
-  margin: 80px 16% 0;
-  border: 1px solid black;
-  width: 50vw;
+.books {
+  background-color: aliceblue;
```

Comment:
- [ ] Kindly remove this background color as the requirements say that your design should match the original template

![image](https://user-images.githubusercontent.com/65068771/180585416-f1c19be1-a4f1-4847-b8da-ec00f36acc5c.png)

Reply:
Hi @andersonlebon, I checked the background color in Zeplin and I changed it as you requested.